### PR TITLE
fix(survey v2): set default value as interface instead of string

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

### CODE REVIEW

File: pkg/input/survey/survey_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package survey_test
```
Comment (ankitm123) [inline pkg/input/survey/survey_test.go:1]: Tests in WIP.

Comment (ankitm123) [reply pkg/input/survey/survey_test.go:1]: Can reproduce the error with the old code with tests, when default is an empty string.

**Comment by ankitm123** [comment]
After this change in jx-helpers:
```
(base) ➜  jx-pipeline git:(main) ✗ ./build/jx-pipeline stop
? Which pipeline do you want to stop:  jenkins-x-plugins/jx-project/PR-464 pr #3
? cancel pipeline jenkins-x-plugins/jx-project/PR-464 pr #3 [? for help] (Y/n) 
error: interrupt
```
Before (Notice how it says empty)
```
(base) ➜  jx-pipeline git:(main) ✗ git stash
Saved working directory and index state WIP on main: 36965c8 Merge pull request #468 from jenkins-x-plugins/pr-5fdab2b2-8094-41d5-84f1-8126f81d8a94
(base) ➜  jx-pipeline git:(main) make build
rm -rf build release dist
CGO_ENABLED=0 GO111MODULE=on go build -ldflags " -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.Version=0.0.149-dev+36965c8 -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.Version=0.0.149-dev+36965c8 -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.Revision='36965c8' -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.Branch='main' -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.BuildDate='20220613-19:51:49' -X github.com/jenkins-x-plugins/jx-pipeline/pkg/cmd/version.GoVersion='1.17.9' " -o build/jx-pipeline cmd/main.go
(base) ➜  jx-pipeline git:(main) ./build/jx-pipeline stop
? Which pipeline do you want to stop:  
? cancel pipeline  [? for help] (Y/n) 
error: interrupt
```

**Review by msvticket — APPROVED** [review]

**Pull request merged**